### PR TITLE
Add retrieval function.

### PR DESCRIPTION
**Pull request by lqq-tj**

### CODE REVIEW

File: themes/apachecon/layouts/search/section.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          <div id="content" class="container">
+            <header class="major">
+              <h2>{{i18n "search"}}</h2>
+          </header>
+            <section class="track">
+            <div class="search" οnlοad="getTestXmlValue();">
+              <input type="text" placeholder="{{i18n "searchContent"}}" id="search-key" />
+              <input type="hidden" name="search_hidden" value="{{i18n "search"}}" id="search_hidden" />
+              <button onclick="search()">
+                <svg t="1583982313567" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="1271"
```
Comment (WillemJiang) [inline themes/apachecon/layouts/search/section.html:26]: Can you check if this image has the copyright?
Can we use it directly here?

File: themes/apachecon/layouts/search/section.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    for (var i = childs.length - 1; i >= 0; i--) {
+      el.removeChild(childs[i]);
+    }
+
+    // xml
+    var language="{{i18n "translation"}}";
+    //var language = document.getElementById("search_hidden").val();
+    //alert(language);
+    xmltext = new XMLHttpRequest;
+    if(language=="中文"){
```
Comment (WillemJiang) [inline themes/apachecon/layouts/search/section.html:81]: It's the hard code, I think you can also use the i18n to specify the index xml reference.
BTW, it's looks strange for me that you use index.xml for Chinese, I think it should be English.

**Review by WillemJiang — COMMENTED** [review]
Just added some comment for this PR.

File: themes/apachecon/layouts/search/section.html (hunk 1 above)
Comment (lqq-tj) [reply themes/apachecon/layouts/search/section.html:26]: I will upload a picture of the search icon again.

**Review by WillemJiang — APPROVED** [review]

**Pull request merged**